config/application.rb

```diff
@@ -1,6 +1,11 @@
 require File.expand_path('../boot', __FILE__)
 
-require 'rails/all'
+# Pick the frameworks you want:
+require "active_record/railtie"
+require "action_controller/railtie"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.